com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -504,33 +504,21 @@ internal void InvokeBehaviourOnGainedOwnership()
             }
         }
 
-        internal void ResetNetworkStartInvoked()
+        internal void InvokeBehaviourNetworkSpawn(Stream stream)
         {
-            if (ChildNetworkBehaviours != null)
+            for (int i = 0; i < ChildNetworkBehaviours.Count; i++)
             {
-                for (int i = 0; i < ChildNetworkBehaviours.Count; i++)
-                {
-                    ChildNetworkBehaviours[i].NetworkStartInvoked = false;
-                }
+                ChildNetworkBehaviours[i].InternalOnNetworkSpawn();
+                ChildNetworkBehaviours[i].OnNetworkSpawn(stream);
             }
         }
 
-        internal void InvokeBehaviourNetworkSpawn(Stream stream)
+        internal void InvokeBehaviourNetworkDespawn()
         {
             for (int i = 0; i < ChildNetworkBehaviours.Count; i++)
             {
-                //We check if we are it's NetworkObject owner incase a NetworkObject exists as a child of our NetworkObject
-                if (!ChildNetworkBehaviours[i].NetworkStartInvoked)
```

Comment:
All these checks did not do anything. They were always set to false before this function was run. The conclusion seems to be that this is legacy code which is not needed.